azurerm/resource_arm_container_group.go

```diff
@@ -108,25 +108,16 @@ func resourceArmContainerGroup() *schema.Resource {
 							}, true),
 						},
 
-						"env_var": {
-							Type:     schema.TypeList,
+						"env_vars": {
```

Comment:
is it worth making this `environment_variables`?

Reply:
Done. Also matches with the CLI.